docs/index.rst

```diff
@@ -51,8 +51,15 @@ The ME-ICA pipeline originally performed both pre-processing and TE-dependent
 analysis of multi-echo fMRI data; however, ``tedana`` now assumes that you're
 working with data which has been previously preprocessed.
 
-.. _ME-ICA: https://github.com/me-ica/me-ica
+For a summary of multi-echo fMRI, which is the technology ``tedana`` builds on, 
```

Comment:
I feel like multi-echo is more of a technique than a technology, but I can be convinced otherwise.

Reply:
I wrote this while looking at this [page](https://www.sciencedirect.com/science/article/pii/S1053811917302410), which says 

> An emerging fMRI technology, multi-echo (ME)-fMRI, makes fMRI more interpretable by enabling the detection of BOLD or artifact origins of acquired signals.

So I was thinking that I could call ME-fMRI "technology." Otherwise, I do agree that multi-echo is a technique used in fMRI. What are your thoughts?

Reply:
Hm, @emdupre @tsalo @dowdlelt is "technology" the preferred etymology? If so then disregard my comment.

Reply:
I vote for "imaging technique".